testing/test-keypad-context-wrapper.tsx

```diff
@@ -9,15 +9,31 @@ import {
 } from "@khanacademy/keypad-context";
 import {MobileKeypad} from "@khanacademy/math-input";
 
-const Footer = (): React.ReactElement => {
+type Props = {
+    children: React.ReactElement;
+    /**
+     * Whether we need to offset the keypad to account for the footer.
+     */
+    hasFooter?: boolean;
```

Comment:
Not entirely convinced this is the best approach to this, but I needed a way to have our test context push the keypad above the footer provided by the ServerItemRendererWithDebugUI while avoiding regressions with the ArticleRendererWithDebugUi.